### PR TITLE
Added --excluded-users and --excluded-terms options

### PR DESCRIPTION
**Pull request by Hyaxia**
Relevant issue - https://github.com/zulip/python-zulip-api/issues/486

Any thing that will be entered in the --exclude string separated by
a ',' will then be checked if it appears in the tweet.
If it does then the tweet wont be sent.

### CODE REVIEW

File: zulip/integrations/twitter/twitter-bot [hunk 1]
```diff
@@ -198,6 +206,16 @@ for status in statuses[::-1][:opts.limit_tweets]:
     text_to_check = [status.text, status.user.screen_name]
     text_to_check.extend(url.expanded_url for url in status.urls)
 
+    # Check that the tweet doesn't contain any terms that
+    # are supposed to be excluded
+    exclude = False
+    for term in excluded_terms:
+        if any(term.lower() in text.lower() for text in text_to_check):
```
Comment (showell) [inline zulip/integrations/twitter/twitter-bot]: A minor optimization here would be to lowercase text_to_check strings outside the loop.

Comment (Hyaxia) [reply zulip/integrations/twitter/twitter-bot]: Will be done, thanks for reviewing

**Comment by timabbott** [comment]
(I did some basic testing on chat.zulip.org, and this does work for filtering worlds from the content of messages; but the main use case we have in mind is filtering messages sent by certain accounts, so we should add that.)

**Comment by timabbott** [comment]
Sorry for the slow review!  This looks great, merged.  Thanks for fixing!

**Pull request closed without merging**